scripts/convert_stable_cascade_lite.py

```diff
@@ -0,0 +1,224 @@
+# Run this script to convert the Stable Cascade model weights to a diffusers pipeline.
+import argparse
+
+import accelerate
```

Comment:
Should be conditioned under `is_accelerate_available()` like the other conversion scripts.